Source/ARTRest.m

```diff
@@ -154,7 +154,11 @@ - (void)executeRequest:(NSMutableURLRequest *)request completion:(void (^)(NSHTT
         }
         if (retries < _options.httpMaxRetryCount && [self shouldRetryWithFallback:request response:response error:error]) {
             if (!blockFallbacks && [request.URL.host isEqualToString:(_prioritizedHost ? _prioritizedHost : [ARTDefault restHost])]) {
```

Comment:
[RSC15b](http://docs.ably.io/client-lib-development-guide/features/#RSC15b) has changed as well, so this condition needs to incorporate this: "or an array of `ClientOptions#fallbackHosts` is provided"